main.go

```diff
@@ -76,13 +79,17 @@ func (e Env) Bucket(svc string) string {
 	if svc == "" {
 		svc = "media"
 	}
-	switch e.Code {
-	case EnvProd:
-		return fmt.Sprintf("prod-%s-unee-t", svc)
-	case EnvDemo:
-		return fmt.Sprintf("demo-%s-unee-t", svc)
-	default:
-		return fmt.Sprintf("dev-%s-unee-t", svc)
+	installationID := e.GetSecret("INSTALLATION_ID")
+	if installationID == "" {
+		installationID = "main"
+		log.Warnf("Using fallback INSTALLATION_ID: %s: ", installationID)
+	}
+	if installationID == "main" {
+		// Preserve original bucket names
+		return fmt.Sprintf("%s-%s-unee-t", e.Stage, svc)
+	} else {
+		// Use INSTALLATION_ID to generate unique bucket name
+		return fmt.Sprintf("%s-%s-%s", e.Stage, svc, installationID)
 	}
 }
 
```

Comment:
@kaihendry the variable `region` looks undefined I think it should be `defaultRegion` instead.
See line 96 and 101.

`func (e Env) SNS(name, region string) string {`

`return fmt.Sprintf("arn:aws:sns:%s:%s:%s", region, e.AccountID, name)`

Reply:
Region is set by arguments passed to the function. It differs in some cases since some things are only present in certain Regions.